onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -419,11 +421,12 @@ class PlannerImpl {
     return Status::OK();
   }
 
-  void GeneratePlanForWeights() {
+  //TODO: Don't generate plan for CPU tensors?
+  Status GeneratePlanForWeights() {
```

Comment:
It's usually helpful to include the 'why' and not just 'what' so the reason for making such a change is captured. e.g. Don't generate plan for CPU tensors because ...